optimizely/optimizely.py

```diff
@@ -20,6 +20,9 @@
 from .config_manager import StaticConfigManager
 from .config_manager import PollingConfigManager
 from .error_handler import NoOpErrorHandler as noop_error_handler
+from .event.event_factory import EventFactory
```

Comment:
is it possible to make it one liner all imports. not sure but you may use * i think.